packages/app/src/elements/import-dialog.ts

```diff
@@ -20,12 +26,24 @@ export class ImportDialog extends Dialog<File, void> {
     @state()
     private _items: VaultItem[] = [];
 
+    @state()
+    private _itemColumns: imp.ImportCSVColumn[] = [];
+
     @query("#formatSelect")
     private _formatSelect: Select<string>;
 
     @query("#vaultSelect")
     private _vaultSelect: Select<Vault>;
 
+    // static styles = [
+    //     ...Dialog.styles,
+    //     css`
+    //         :host {
+    //             --pl-dialog-max-width: 40em;
+    //         }
+    //     `,
+    // ];
```

Comment:
@MaKleSoft So I tried this, but the dialog got transparent and full-width. What's wrong (note I also might not need this, but would like to understand, regardless)?

Reply:
Hm, working fine for me...

Reply:
Eh, actually worked this time. I might've had something else going on.